### ISSUE TITLE
ai

### ISSUE THREAD

**Issue by romanskyitymur-ctrl** [romanskyitymur-ctrl/Tymur#2]
from sklearn.feature_extraction.text import CountVectorizer
from sklearn.linear_model import LogisticRegression

# 1. Дані для навчання нашого бізнес-ШІ
reviews_database = [
    "Мені все сподобалося, супер якість", 
    "Жахливий сервіс, довго чекав, все погано",
    "Дякую, чудовий товар!", 
    "Огидно, зламалося одразу, не купуйте"
]
labels =  # 1 - задоволений, 0 - злий

vectorizer = CountVectorizer()
X_train = vectorizer.fit_transform(reviews_database)

ai_business_core = LogisticRegression()
ai_business_core.fit(X_train, labels)

# 2. ТЕСТ СИСТЕМИ: Приходить новий відгук від покупця
new_review = ["Мені прислали брак, я буду скаржитися!"]
X_test = vectorizer.transform(new_review)
sentiment = ai_business_core.predict(X_test)

# 3. ЛОГІКА АРХІТЕКТОРА: Прийняття бізнес-рішення
print("--- СИСТЕМА АВТОМАТИЗАЦІЇ БІЗНЕСУ ШІ ---")
if sentiment == 1:
    print("🤖 Статус відгуку: 😊 Позитивний.")
    print("🤖 Дія системи: Нарахувати клієнту +50 бонусів подяки. Надіслати лист: 'Дякуємо, що ви з нами!'")
else:
    print("🤖 Статус відгуку: 😡 КРИЗА! Негативний відгук!")
    # Рахуємо знижку за допомогою математики (Урок 2)
    base_discount = 10
    compensation = 15
    final_discount = base_discount + compensation
    print(f"🤖 Дія системи: 🚨 Автоматично активовано промокод на знижку {final_discount}% для утримання клієнта!")
